### PR TITLE
Shows password protected visualizations

### PR DESCRIPTION
**Pull request by javierarce**
Password protected visualizations require to get an array with the `auth_tokens`. This PR solves that.

@javisantana: can you :eyes:, please?

### CODE REVIEW

**Comment by javisantana** [comment]
looks good but I would not use viz.json online, that is something that will fail at some point :(

**Comment by javierarce** [comment]
retest this please

**Comment by javierarce** [comment]
You're right. I keep postponing a fix for that. 

I'm assigning myself this ticket: https://github.com/CartoDB/cartodb.js/issues/404

**Comment by javierarce** [comment]
@javisantana: can you review [my last change](https://github.com/CartoDB/cartodb.js/commit/aee06ea9c1963228b99f1d495328b80066f69157), please?

File: test/spec/core/image.spec.js [hunk 1]
```diff
@@ -433,4 +433,35 @@ describe("Image", function() {
...
+
+    StaticImage.prototype.load = function(vizjson, options) {
```
Comment (javisantana) [inline test/spec/core/image.spec.js]: why is this needed?

Comment (javierarce) [inline test/spec/core/image.spec.js]: To test the vizjson response without making a real call. Do you prefer to use other method?

Comment (javisantana) [inline test/spec/core/image.spec.js]: you could test using the layergroup version of contrcutor, right?

Comment (javierarce) [inline test/spec/core/image.spec.js]: Yes, that wouldn't test the regular method which has the majority of the code.

Comment (javierarce) [inline test/spec/core/image.spec.js]: bump

**Comment by javisantana** [comment]
:es:
